tests/unit/scripting.tcl

```diff
@@ -1,38 +1,65 @@
 foreach is_eval {0 1} {
+foreach script_compatibility {server redis} {
+
+if {$script_compatibility eq "server"} {
+    proc replaceLuaRedisAPI {args} {
+        set new_string [regsub -all {redis\.} [lindex $args 0] {server.}]
+        return lreplace $args 0 0 $new_string
+    }
+
+    proc get_api_name {} {
+        # TODO: Replace this with server when it is working
+        return "redis"
+    }
+} else {
+    proc replaceLuaRedisAPI {args} {
+        return lreplace $args 0 0 [lindex $args 0]
```

Comment:
```suggestion
        return $args
```
Doesn't work and I'm not entirely sure why. Maybe someone who understands lua better can understand this. It was wrapping the response.

Reply:
Oh, I think I figured it out, I think it should be
```suggestion
        return {*}$args
```